src/pio/mod.rs

```diff
@@ -1,28 +1,21 @@
-use std::{
-    collections::{HashMap, HashSet},
-    convert::{TryFrom, TryInto},
-    ffi::OsStr,
-    fs::{self, File},
-    io::{Read, Write},
-    path::{Path, PathBuf},
-    process::{Command, Output, Stdio},
-};
+pub mod project;
+
+use std::collections::{HashMap, HashSet};
+use std::convert::{TryFrom, TryInto};
+use std::ffi::OsStr;
+use std::fs::{self, File};
+use std::io::{Read, Write};
+use std::path::{Path, PathBuf};
+use std::process::{Command, Output, Stdio};
 
 use anyhow::*;
 use log::*;
-
+use serde::de::DeserializeOwned;
+use serde::{Deserialize, Serialize};
 use tempfile::*;
 
-use serde::{de::DeserializeOwned, Deserialize, Serialize};
-
-pub mod bindgen;
-pub mod bingen;
-pub mod cargo;
-pub mod project;
-pub mod symgen;
-
 const INSTALLER_URL: &str = "https://raw.githubusercontent.com/platformio/platformio-core-installer/master/get-platformio.py";
-const INSTALLER_BLOB: &[u8] = include_bytes!("resources/get-platformio.py.resource");
+const INSTALLER_BLOB: &[u8] = include_bytes!("../resources/get-platformio.py.resource");
```

Comment:
Suggestion: how about moving all PlatformIO-specific resources in a `resources` subdirectory of `/pio/`, and refering them from there, rather than doing the `../` trick?